db/migrate/20190422130439_change_status_to_enum_in_requests.rb

```diff
@@ -0,0 +1,27 @@
+class ChangeStatusToEnumInRequests < ActiveRecord::Migration[5.2]
+  def up
+    add_column :requests, :status_new, :integer, default: 0
+
+    Request.where(status: 'Active').update_all status_new: 0
+    Request.where(status: 'Fulfilled').update_all status_new: 2
```

Comment:
This is a minor change -- we've gotten bitten by this before; could you add:

```ruby
class Request < ApplicationRecord; end
```
to the very beginning of the file? 

On the offchance `Request` is later renamed to something else, this will ensure that the migration is able to be performed at this point.

Thank you!

Reply:
Sure!

Reply:
Thank you! :100:

Reply:
@armahillo Done! Thank you